src/tests/mod.rs

```diff
@@ -138,6 +139,42 @@ fn strings() {
     );
 }
 
+#[test]
+fn arc() {
```

Comment:
This test is pretty fragile as-is, it'll fail on any target that doesn't have 64bit pointers. Instead of hard-coding pointer sizes, this should use `size_of::<usize>()`

Reply:
Good point. I updated the test